topics/reference/easy/simple-swap.tex

```diff
@@ -23,4 +23,7 @@
 \end{verbatim}
 In the main method, \texttt{x} and \texttt{y} won't actually be swapped.
 Within \texttt{swap}, we can change what \texttt{a} and \texttt{b} point to, but we can't change the variables that were declared in \texttt{main}. We can fix this by either in-lining the \texttt{swap} functionality in the \texttt{main} method or returning and reassigning the swapped values using an object.
+
+\textbf{Meta:} Emphasize "Golden Rule of Equals" (pass-by-value). The bits are copied over.
```

Comment:
Quotes in TeX should be specified using \`\` (two ticks) and '' (two apostrophes) instead of ".